src/apps/testapps/testBBox.c

```diff
@@ -219,4 +219,22 @@ SUITE(BBox) {
         t_assert(bboxIsTransmeridian(&bboxTransmeridian),
                  "Transmeridian bbox is transmeridian");
     }
+
+    TEST(bboxEquals) {
+        BBox bbox = {1.0, 0.0, 1.0, 0.0};
+        BBox north = bbox;
+        north.north += 0.1;
+        BBox south = bbox;
+        south.south += 0.1;
+        BBox east = bbox;
+        east.east += 0.1;
+        BBox west = bbox;
+        west.west += 0.1;
+
+        t_assert(bboxEquals(&bbox, &bbox), "Equals self");
+        t_assert(!bboxEquals(&bbox, &north), "Equals different north");
```

Comment:
Nit: I'd have the assertion messages here express the desired behavior, e.g. `Does not equal different north`